### DISCUSSION TITLE
[Help Wanted] How do you update Standard Notes manually on MacBook? Automatic updates don't work..

### DISCUSSION THREAD

**Discussion by ballindigital** [standardnotes/forum#2935] [General] [answered]
Automatic updates don't appear to be working on my MacBook.

How do I update Standard Notes manually?

I would appreciate specific instructions as it just says to download and install.

What does this mean?

I am using Version 3.20.0 (3.20.0)

<img width="253" alt="Screenshot 2023-01-10 at 16 57 25" src="https://user-images.githubusercontent.com/122388589/211614417-27aed9b0-3ba5-4dc8-af5e-0f0e29db156a.png">

**Comment by dghetherington** [comment]
You could download and run the installer from this site - https://standardnotes.com/download

**Comment by ballindigital** [comment]
Thanks - when it says to replace the old installation won't it delete everything?

**Comment by dghetherington** [comment, marked as answer]
I did it on mine today, after installing the newer one I had to sign in again, but all of my notes synced from the server

to double check that you won't lose anything, you could log into the web app - https://app.standardnotes.com/ - and if all your notes are there then you should be fine, you may need to log in using the little 'head' icon in the bottom left first.

**Comment by ballindigital** [comment]
Thanks bro - it worked.. I didn't have to sign back in. It was all synced up.

Hopefully it will auto update now :)

**Comment by dghetherington** [comment]
great, thanks for the confirming.
